### PR TITLE
add rishabh062

### LINKED ISSUE #390: Rishabh Dwivedi
Screenshot (14)

### PR DESCRIPTION
**Pull request by Rishabh062**
resolve #390

### CODE REVIEW

File: data.json [hunk 1]
```diff
@@ -497,6 +497,12 @@
     "imageUrl": "https://user-images.githubusercontent.com/66197682/105803643-eaf33180-5fc3-11eb-8f97-42de7cee0f05.png",
     "issueNumber": 279
   },
+  {
```
Comment (AllanRegush) [inline data.json:500]: Please Add yourself to the bottom

**Review by AllanRegush — APPROVED** [review]
Looking Good :+1: Just a few adjustments.

**Comment by AllanRegush** [comment]
Please do not make another PR to fix. Please adjust this one. If you need help with that let me know. :nerd_face:

File: data.json (hunk 1 above)
Comment (Rishabh062) [reply data.json:500]: Name = Rishabh Dwivedi
id= rishabh062
issue =390

Comment (Rishabh062) [reply data.json]: Done

**Review by AllanRegush — APPROVED** [review]
Looks Good :+1: Thank you

**Pull request merged**